docs/developer-guide/unit-testing.md

```diff
@@ -338,14 +294,20 @@ If the test your are creating belongs to a check that uses more than one provide
 
 ```python
 with mock.patch(
-    "prowler.providers.aws.lib.audit_info.audit_info.current_audit_info",
-    new=mock_audit_info,
+    "prowler.providers.common.common.get_global_provider",
```

Comment:
We have to point that the `set_mocked_aws_provider` has a lot of different attributes to customize the AWS provider for the check we are testing.

Reply:
Totally agree